src/components/special/_rounded-diagonal.less

```diff
@@ -0,0 +1,19 @@
+.rounded-diagonal-btn {
+  background: #7c26c2;
+  font: bold 24px "Syne";
```

Comment:
remove font-family "Syne" as this is set in the website but shouldn't be set in sbuttons.css

Reply:
I've set that font to overlap to cursive-font which doesn't match with the button theme. And 'retro button' .less file is also having the custom font.

Reply:
In retro button the font is imported in the Less file and it is necessary to the button's design. Syne is not imported in the Less file it's just the font of the website. If you want to change the font-family to Syne then set it in `assets/css/index.css` as it is not required for the button itself.